setup.py

```diff
@@ -5,9 +5,8 @@
 from distutils.core import setup
 import setuptools
 
-def read(fname):
-    """From an_example_pypi_project (https://pypi.python.org/pypi/an_example_pypi_project). """
-    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+with open("README.md", "r") as fh:
+    long_description = fh.read()
```

Comment:
FYI: a small code improvement — following code example in https://packaging.python.org/tutorials/packaging-projects/#creating-setup-py